### PR TITLE
Add unwrapped tests

### PR DESCRIPTION
**Pull request by jjshoots**
This adds unwrapped tests for both Gym and AEC environments.

### CODE REVIEW

File: test/gym_unwrapped_test.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        env = frame_skip_v0(env, 2)
+
+    # everything else
+    env = clip_reward_v0(env, lower_bound=-1, upper_bound=1)
+    env = delay_observations_v0(env, 2)
+    env = sticky_actions_v0(env, 0.5)
+    env = nan_random_v0(env)
+    env = nan_zeros_v0(env)
+
+    assert env.unwrapped == base_env, f"Failed to unwrap {env}"
```
Comment (benblack769) [inline test/gym_unwrapped_test.py:66]: Could you do something along the lines of `env.unwrapped.__class__ == DummyEnv`?

File: test/aec_unwrapped_test.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    if action_homogenizable(env, agents):
+        env = pad_action_space_v0(env)
+
+    env = clip_reward_v0(env, lower_bound=-1, upper_bound=1)
+    env = delay_observations_v0(env, 2)
+    env = sticky_actions_v0(env, 0.5)
+    env = nan_random_v0(env)
+    env = nan_zeros_v0(env)
+
+    assert env.unwrapped == base_env, f"Failed to unwrap {env}"
```
Comment (benblack769) [inline test/aec_unwrapped_test.py:131]: Could you do something along the lines of `env.unwrapped.__class__ == DummyEnv`?

Comment (jjshoots) [reply test/aec_unwrapped_test.py:131]: Gotcha, will do.

**Comment by benblack769** [comment]
Looks good to me.

**Pull request merged**